.dockerignore

```diff
@@ -54,6 +54,7 @@
 /canopy/**/coverage
 /canopy/**/build
 /canopy**/dist
+/canopy/**/node_modules
```

Comment:
Isn't this already accomplished by line 50?

Reply:
I thought so, but I thought I was getting node_modules when I built